src/commands.ts

```diff
@@ -678,8 +677,28 @@ export const insertList =
       return true
     } else {
       // no list found, create new list
-      const command = wrapInList(type, { listStyleType: style })
-      return skipCommandTracking(command)(state, dispatch)
+
+      const { selection } = state
+      let tr = state.tr
+
+      return wrapInList(type, { listStyleType: style })(state, (tempTr) => {
+        // if we dispatch all steps in this transaction track-changes-plugin will not be able to revert ReplaceAroundStep
```

Comment:
we probably need to file a ticket for track-changes plugin to support that case properly so we won't have to run a transaction for each step.

Reply:
I would say we still should think about on how to make tc-plugin properly handle such cases. I mean if breaking it into several transaction does the trick the least we could do is to treat such steps as if they were in separate transaction but inside the tc plugin. In any case that shouldn't be blocking this work and can be filed separately if preferred to be handled so.